Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -228,18 +228,20 @@ function _flush(rootNode: AnimatedValue): void {
   animatedStyles.forEach(animatedStyle => animatedStyle.update());
 }
 
-type TimingAnimationConfig =  AnimationConfig & {
+type TimingAnimationConfig = AnimationConfig & {
   toValue: number | AnimatedValue | {x: number, y: number} | AnimatedValueXY,
   easing?: (value: number) => number,
   duration?: number,
   delay?: number,
+  numLoops?: number,
```

Comment:
I think this should be called `iterations` since that the name of the parameter in `Animated.loop`, so `numLoops` -> `iterations`.